.github/workflows/build_image_test.yaml

```diff
@@ -0,0 +1,19 @@
+name: build docker image
```

Comment:
Stick to a convention. You have `jira-integration` but `build_image_test`. Either use snake-case or underscores as you like.